src/Components/Facility/PatientNotesSlideover.tsx

```diff
@@ -199,32 +199,36 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
             disableEdit={!patientActive}
           />
           <div className="relative mx-4 flex items-center">
-            <AutoExpandingTextInputFormField
-              id="doctor_notes_textarea"
-              maxHeight={160}
-              rows={1}
-              name="note"
-              value={noteField}
-              onChange={(e) => setNoteField(e.value)}
-              className="grow"
-              errorClassName="hidden"
-              placeholder="Type your Note"
-              disabled={!patientActive}
-              onFocus={() => setFocused(true)}
-              onBlur={() => setFocused(false)}
-            />
-            <ButtonV2
-              id="add_doctor_note_button"
-              onClick={onAddNote}
-              border={false}
-              className="absolute right-2"
-              ghost
-              size="small"
-              disabled={!patientActive}
-              authorizeFor={NonReadOnlyUsers}
-            >
-              <CareIcon className="care-l-message text-lg" />
-            </ButtonV2>
+            <div className=" w-full px-12">
+              <AutoExpandingTextInputFormField
+                id="doctor_notes_textarea"
+                maxHeight={160}
+                rows={1}
+                name="note"
+                value={noteField}
+                onChange={(e) => setNoteField(e.value)}
+                className="grow"
+                errorClassName="hidden"
+                placeholder="Type your Note"
+                disabled={!patientActive}
+                onFocus={() => setFocused(true)}
+                onBlur={() => setFocused(false)}
+              />
+            </div>
+            <div className="flex items-center">
+              <ButtonV2
+                id="add_doctor_note_button"
+                onClick={onAddNote}
+                border={false}
+                className="absolute right-2"
+                ghost
+                size="small"
+                disabled={!patientActive}
+                authorizeFor={NonReadOnlyUsers}
+              >
+                <CareIcon className="care-l-message text-lg" />
+              </ButtonV2>
+            </div>
```

Comment:
Why was this changed? Totally unrelated to the issue being solved right?

Reply:
To Make the layout better... Since very soon, upload icon will also be added and so thought making a layout before would be better. Just rearrangement, thats it. Will remove if you say.

Reply:
Unrelated to the issue being solved by this PR. Feel free to file a separate issue if so.

Reply:
Should I remove them and file separate issue for this as well?

Reply:
Just saw this was addressed in another PR. Need not file an issue

Reply:
Yeah... actually that was addressed before this, both by me, therefore, it happened so.

Reply:
Also changed the title as was instructed.

Reply:
@Sulochan-khadka you've not pushed any changes plus the PR title still doesn't make sense. Refer the above link and see other PR's for example...